src/S3Control/EndpointArnMiddleware.php

```diff
@@ -69,26 +72,30 @@ class EndpointArnMiddleware
      */
     public static function wrap(
         Service $service,
-        $region,
-        array $config
-
-    ) {
-        return function (callable $handler) use ($service, $region, $config) {
-            return new self($handler, $service, $region, $config);
+                $region,
+        array   $config,
```

Comment:
looks a bit odd having only one of them have the type here, and the indentation is off

Reply:
The diff makes this look even stranger, but those types were there before.  `Service` has a type as well